text/0000-embedded-refract.md

```diff
@@ -0,0 +1,361 @@
+- Start Date: 2015-08-29
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This document outlines a new serialization format for embedding Refract.
+
+# Motivation
+
+1. Our current formats either look like XML, the DOM, or Lisp
+1. People already "get" JSON
+1. Other serializations we have are verbose
+
+# Detailed design
+
+## Overview
+
+For this serialization, there are two basic concepts:
+
+1. There is a reserved property name of `refract`
+1. There is a Refract Object with properties:
+    - element
+    - meta
+    - attributes
+    - content
```

Comment:
Is there any guidance on how to handle name collisions? What would be the expected behavior of an implementation that could not produce this serialization because of a pre-existing `refract` property?